infra/token-transfer-server/src/scripts/execute_transfers.js

```diff
@@ -1,153 +0,0 @@
-/*
```

Comment:
Was this file deleted by mistake ? I was thinking we would schedule this script as a cron to execute the transfers enqueued... :)

Reply:
Yep will restore :grin: